introToConnectionMethods/README.md

```diff
@@ -0,0 +1,45 @@
+# Overview
+Gain an understanding of the different connection methods you can use to connect to an Astra DB database. 
+
+**In this guide, we'll learn**
+- The different types of connection methods and when to use them
+- The basics on using an API to connect to Astra DB
+- The basics on using a Driver to connect with Astra DB
+- The basics on using gRPC clients to connect with Astra DB
+
+**Prerequisites**
+- You may want to [create a database](https://docs.datastax.com/en/astra-serverless/docs/) before jumping in, but you don't have to. 
+
+
+## 1  Different Connection Methods, and when to use 'em
+Astra DB has a bunch of different connection methods to take advantage of. APIs are great for getting started quickly. Drivers are tried and tested – Cassandra battle-tanks. gRPC represents the future, and combine some of the best things you get from using APIs and Drivers – they're cloud-native out of the box. And finally, there's a bunch of SDKs available for you. Want to see them all in practice? Go ahead and create a database to get started.
+
+```<<createDatabase>>```
```

Comment:
Oops. If you want to include an action, remove the ticks and simply insert <<createDatabase>> directly into the markdown. The ticks were for illustration purposes only.